cmd/core-data/res/configuration.toml

```diff
@@ -52,5 +52,6 @@ Protocol = 'tcp'
 Host = '*'
 Port = 5563
 Type = 'zero'
+Topic = 'events'
```

Comment:
Needs to be added to `cmd/core-data/res/docker/configuration.toml` also

Reply:
will do